app/commands/tooling_job/process.rb

```diff
@@ -29,7 +29,7 @@ def call
           attrs["iteration_uuid"],
           attrs["execution_status"],
           "Nothing to report", # TODO
-          download_file(attrs["output"]["representation.txt"]),
+          JSON.parse(download_file(attrs["output"]["representation.txt"])),
```

Comment:
Without this, the data in the DB is stored as a string with double quotes around it.

Reply:
Hmmm, does that not seem wrong to you? I suspect this means that the representation.txt is getting uploaded with the extra quotes incorrectly in the first place?

Reply:
It does. It is on my "todo" list, but I wanted to get the PR's up for Mark (mjs) to be able to continue. I'll dig a bit deeper into this.